src/electionguard_cli/steps/shared/election_record_step.py

```diff
@@ -0,0 +1,82 @@
+from shutil import make_archive
+from os.path import splitext, dirname, basename
+from tempfile import TemporaryDirectory
+from click import echo
+from electionguard import to_file
+from electionguard.constants import get_constants
+
+from electionguard_cli.cli_models import BuildElectionResults, E2eSubmitResults
+from electionguard_cli.cli_models.e2e_decrypt_results import E2eDecryptResults
+from electionguard_cli.cli_models.e2e.e2e_inputs import E2eInputs
+
+from electionguard_tools.helpers.export import export_record
+
+from .cli_step_base import CliStepBase
+
+
+class ElectionRecordStep(CliStepBase):
```

Comment:
Not much of this class actually changed, I just moved a lot of steps around and github seems to have failed to recognize the moves